iset.mm

```diff
@@ -65414,6 +65414,27 @@ all reals (greatest real number) for which all positive reals are greater.
       PLZJVRVTXEBCVCVEVKVPVFVGQR $.
   $}
 
+  ${
+    $d A f g h $.  $d B f g h $.  $d C f g h $.  $d D f g h $.
+    $( Dominance law for disjoint union.  (Contributed by BJ and Jim Kingdon,
+       25-Jul-2022.) $)
```

Comment:
The key step is casef1, which I contributed, but the proof of djudom is really "contributed by JK". Or you can leave it as is. As you prefer.

Reply:
I wasn't basing it on casef1 because that has its own contributed line. I was basing it on you having written the informal proof in a comment here.

I guess I'll go with the simpler Contributed line, but still you should know that your comment, even though you thought you were just saying an obvious thing which I already knew, was very helpful. Without it I would have had to go back and read old messages, remember what `case` has to do with `djudom`, etc.